src/connection.cpp

```diff
@@ -1145,7 +1208,7 @@ namespace oxen::quic
 
         if (uint64_t app_err_code = context->stream_open_cb ? context->stream_open_cb(*stream) : 0; app_err_code != 0)
         {
-            log::info(log_cat, "stream_open_callback returned error code {}, closing stream {}", app_err_code, id);
+            log::warning(log_cat, "stream_open_callback returned error code {}, closing stream {}", app_err_code, id);
```

Comment:
I don't think this should be a log::warning, because this is something the callback is supposed to be able to do to close a stream.  (Or rather: to the extent that is is something warnable is up to the application, not libquic).  Warnings, rather, should be for "something went wrong" messages.